pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -669,8 +674,14 @@ const (
 	// or the specified namespace was not present.
 	NamespaceCreatedByProjectController = "namespace.gardener.cloud/created-by-project-controller"
 
-	// DefaultVpnRange is the default network range for the vpn between seed and shoot cluster.
-	DefaultVpnRange = "192.168.123.0/24"
+	// DefaultVPNRange is the default network range for the VPN between seed and shoot cluster (and the first VPN tunnel in HA case).
+	DefaultVPNRange = "192.168.123.0/24"
+	// DefaultHAVPNRangeBonding is the network range for the VPN between seed and shoot cluster if high availability is enabled.
+	DefaultHAVPNRangeBonding = "192.168.122.0/24"
+	// DefaultHAVPNRange1 is the network range for the second VPN tunnel between seed and shoot cluster if high availability is enabled.
+	DefaultHAVPNRange1 = "192.168.124.0/24"
+	// DefaultHAVPNRange2 is the network range reserved for a third VPN tunnel between seed and shoot cluster if high availability is enabled.
+	DefaultHAVPNRange2 = "192.168.125.0/24"
```

Comment:
What if existing shoot clusters already use these ranges for their networks?

Reply:
The ranges are validated for the shoot defaults of seed objects and in the shoot controller of the scheduler only. It does not produce validation errors for existing shoots.
In the unlikely case of an existing shoot cluster is using these network ranges, enabling HA can potentially result in strange network issues. The validation can avoid such a situation at least for new shoots.
@ScheererJ Anything to add?

Reply:
Yes, new shoots will not be able to get created, but shoots might already exist with these ranges (not sure why you consider this "unlikely").

Reply:
VPN2 `0.13.0` release ready and integrated. Constants, tests and docu have been adjusted to reflect the latest changes to the VPN network ranges.

/unhold